pkgs/by-name/sq/squirreldisk/package.nix

```diff
@@ -26,92 +32,75 @@ let
     rev = "v${version}";
     hash = "sha256-As2nvc68knjeLPuX0QLBoybj8vuvkpS5Vr+7U7E5CjA=";
   };
-  frontend-build = buildNpmPackage {
-    inherit version src;
-    pname = "squirreldisk-ui";
 
-    npmDepsHash = "sha256-Japcn0KYP7aYIDK8+Ns+mrnbbAb0fLWXHIV2+yltI6I=";
-
-    packageJSON = ./package.json;
-    postBuild = ''
-      cp -r dist/ $out
-    '';
-    distPhase = "true";
-    dontInstall = true;
-
-    patches = [
-      # Update field names to work with pdu versions >=0.10.0
-      # https://github.com/adileo/squirreldisk/pull/47
-      ./update-pdu-json-format.patch
-    ];
-  };
-in
-rustPlatform.buildRustPackage rec {
-  inherit version src pname;
-
-  sourceRoot = "${src.name}/src-tauri";
+  cargoRoot = "src-tauri";
+  buildAndTestSubdir = "src-tauri";
 
   useFetchCargoVendor = true;
   cargoHash = "sha256-dFTdbMX355klZ2wY160bYcgMiOiOGplEU7/e6Btv5JI=";
 
-  # copy the frontend static resources to final build directory
-  # Also modify tauri.conf.json so that it expects the resources at the new location
-  postPatch = ''
-    mkdir -p frontend-build
-    cp -r ${frontend-build}/* frontend-build
+  npmDeps = fetchNpmDeps {
+    name = "squirreldisk-${version}-npm-deps";
+    inherit src;
+    hash = "sha256-Japcn0KYP7aYIDK8+Ns+mrnbbAb0fLWXHIV2+yltI6I=";
+  };
 
-    substituteInPlace tauri.conf.json --replace-fail '"distDir": "../dist"' '"distDir": "./frontend-build"'
+  patches = [
+    # Update field names to work with pdu versions >=0.10.0
+    # https://github.com/adileo/squirreldisk/pull/47
+    ./update-pdu-json-format.patch
+  ];
 
-    # Copy pdu binary from nixpkgs, since the default packaged binary has issues.
-    cp ${parallel-disk-usage}/bin/pdu bin/pdu-${stdenv.hostPlatform.config}
+  postPatch = ''
+    # Use pdu binary from nixpkgs instead of the vendored prebuilt binary
+    rm src-tauri/bin/pdu-*
```

Comment:
This line was added so that we don't accidentally use any of the bundled binaries.
I don't know how it could possibly happen, but I think it's better like this.